src/app/gallery/page.js

```diff
@@ -1,5 +1,60 @@
+import React from "react";
+import Image from "next/image";
+import Title from "@/components/Title";
+import Subtitle from "@/components/Subtitle";
+import Hoverable from "@/components/gallery/Hoverable";
+import { quarterlyEvents } from "@/data/gallery/QuarterlyEvents";
+import Tabling1 from "@/public/images/tabling/tabling_1.webp";
+import Tabling2 from "@/public/images/tabling/tabling_2.webp";
+import Tabling3 from "@/public/images/tabling/tabling_3.webp";
+import PastEvent from "@/public/images/past/past_3.webp";
+
 const Page = () => {
-  return <div>gallery</div>;
+  return (
+    <div className="bg-green-100 min-h-screen p-7 w-full flex flex-col items-center">
+      <div className="flex justify-center mb-10">
+        <Title text="Gallery" food="chip" color="biscuits-grey-100" />
+      </div>
+
+      <div className="mb-14">
+        <div className="mb-10 flex justify-center">
+          <Subtitle text="Food Tabling" color="biscuits-green-200" />
```

Comment:
```suggestion
          <Subtitle text="Food Tabling" color="text-biscuits-green-200" />
```

repeat for other subtitles